### PR TITLE
Sections

### PR DESCRIPTION
**Pull request by Alondra87**
# Portfolio: setup and mobile version skeleton

> For this project, I used the HTML&CSS linter
> There are the follow sections:
   -The toolbar
   -The headline section 
   -Middle section
   -About myself section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,26 +21,170 @@ <h2 class="mylogo">
...
+            <li class="icon_languaje"><a class="link_leng" href="#"><img class="icon_leng"  src="./css/html5.png" alt="html5">HTML</a></li>
+            <li class="icon_languaje"><a class="link_leng" href="#"><img class="icon_leng" src="./css/css3.png" alt="css3">CSS</a></li>
+        </ul>
+    </details>
+    <details class="list_detail">
+        <summary>Frameworks</summary>
+    </details>
+    <details class="list_detail">
+    <summary>Skills</summary> 
+    </details>
```
Comment (Danieldotcomcoder) [inline index.html]: Kindly include technology categories ("Languages", "Frameworks" and "Skills") all in a `ul` tag each one of them in a `li` tag that will make your code more professional 😃✔

File: css/styles.css [hunk 2]
```diff
@@ -79,14 +133,171 @@ li {
...
+  color: #396df2;
+}
+
+.button_project {
+  width: 295px;
+  height: 72px;
+  top: 260px;
+  padding-left: 16px;
+  border: none;
+}
```
Comment (Danieldotcomcoder) [inline css/styles.css]: Kindly add the button interactions from the Figma template (hover, active ) to the 'see project' button 😃✔😃

image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Alondra87, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Alondra87** [comment]
Thank you for your comments on those important points.

**Review by KayLemba — APPROVED** [review]
Hi @Alondra87 

Good job,
Congratulations  your project has been approved . There is nothing more to say than its time to merge your work. 🥇  Keep  up the good work.

## Highlights
- App is working as intended
- No linter errors 
- Descriptive Pull Request

## Required Changes ♻️
N/A

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0Oa

**Pull request merged**